src/controllers/observation-controller.ts

```diff
@@ -54,11 +56,38 @@ export default class ObservationController extends AbstractController {
 
   private getObservations = async (req: RequestWithPageParams, res: Response, next: NextFunction): Promise<void> => {
     try {
+      const { lang = 'desc_eng' }: { lang: Locale } = req.query;
       const paramsSearch = parsePageParams(req.query);
       const paramsAggregation = parseWhereParams(req.query, req.user);
       const observations = await this.observations.findAndCount(Object.assign(paramsSearch, paramsAggregation));
+
+      const content = observations[0].map(obs => {
+        // sanitaze user's data
+        const finder = Object.assign({}, obs.finder.sanitizeUser(), { id: obs.finder.id });
+        // transform 'observation'
+        const observation = Object.entries(obs)
+          // clear 'filter' field
+          .filter(([ObservationField]) => ObservationField !== 'ring')
```

Comment:
probably ring can be useful on client side as pointer to related entity, simply speaking as link to page. at least it needed at single entity

Reply:
Ok, below I have seen that it's returned